src/sagemaker/lineage/artifact.py

```diff
@@ -335,6 +386,17 @@ def list(
             sagemaker_session=sagemaker_session,
         )
 
+    def get_artifacts(self, s3_uri: str):
```

Comment:
`get_artifacts` -> `s3_uri_artifacts` ? 

Python doesn't support function overloading so we have to be careful about naming these too generically.